great_expectations/rule_based_profiler/config/base.py

```diff
@@ -155,6 +155,7 @@ def __init__(
         name: str,
         class_name: str,
         module_name: Optional[str] = None,
+        json_serialize: Optional[bool] = False,
```

Comment:
Why `Optional[bool]`? If the arg is not required, could we not get away with `bool` and the default value?

Reply:
@cdkini Fixed (good catch).  Thanks.